maxprocs/maxprocs.go

```diff
@@ -60,6 +61,13 @@ func Logger(printf func(string, ...interface{})) Option {
 	})
 }
 
+// Min sets the minimum GOMAXPROCS value that will be used.
+func Min(n int) Option {
+	return optionFunc(func(cfg *config) {
+		cfg.minGOMAXPROCS = n
```

Comment:
Might be worth either handling n<1 or noting in the docs that passing n<1 leaves `GOMAXPROCS` unchanged.

Reply:
Good idea, will note that `GOMAXPROCS` < 1 will have no affect.